Tests/Clappr_Tests/Classes/Plugin/Container/ContainerTests.swift

```diff
@@ -10,6 +10,11 @@ class ContainerTests: QuickSpec {
             var container: Container!
 
             describe("#Init") {
+
+                beforeEach {
+                    Loader.shared.resetPlaybacks()
```

Comment:
As far as I can see the bigger problem in these tests was this missing logic. Maybe we can discuss in our next tech session about a better approach to avoid this mistake in the future.

Reply:
Yes! I agree completely!